// *****************************************************************************
//
// This file is part of a MASA library or program.
// Refer to the included end-user license agreement for restrictions.
//
// Copyright (c) 2010 MASA Group
//
// *****************************************************************************

package MsgsDispatcherToClient;

message MsgServicesDescription {
    repeated string services = 1;
}

message MsgDispatcherToClient {
    message Content {
        optional MsgServicesDescription services_description = 1;
    }
    required Content message = 1;
}